### ISSUE TITLE
MySensor.h where is it?

### ISSUE THREAD

**Issue by Ricardosgeral** [empierre/arduino#32]
Hi all
I'm trying to read an MQ131 but i have an error (missing library)

Where do i get this library
#include <MySensor.h> 

I only found in MySensors library  a file named "MySensors.h" with an s at the end.

thanks

**Comment by empierre** [comment]
Hello,

  it is a communication library for Arduino and alike, see here: https://www.mysensors.org/

**Comment by Ricardosgeral** [comment]
yes i suspected, but even so there is no "MySensor.h" but a "MySensors.h"

So a bug in this code?

**Comment by empierre** [comment]
maybe an old lib reference for I didn't implemented above 1.4 yet.
